project-1/pet-store-mvc/controllers/PetControllers.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+ini_set('display_errors', 1);
+
+include_once 'models/model.php';
+
+class PetController
+{
+    private $model;
+    public function __construct($conn)
+    {
+        $this->model = new PetModel($conn);
+    }
+
+    public function displaySpeciesType()
+    {
+        $species = $this->model->getSpeciesType();
+        include "views/petView.php";
+        // return $pets;
+    }
+
+    public function petForm()
+    {
+        include "views/petForm.php";
+    }
+
+    public function addSpeciesType()
+    {
+        $type = $_POST['type'];
+
+        if (!$type) {
+            echo "Please fill out all fields";
+            $this->petForm();
+            return;
+        } elseif ($this->model->insertSpeciesType($type)) {
+            echo "Pet type added successfully: $type";
+        } else {
+            echo "Error adding pet type";
+            $this->petForm();
+        }
+        $this->displaySpeciesType();
+    }
+
+    public function updateSpeciesType($id, $new_pet_species_type)
+    {
+        return $this->model->updateSpeciesType($id, $new_pet_species_type);
+    }
+
+    public function deleteSpeciesType($id)
+    {
+        return $this->model->deleteSpeciesType($id);
+    }
```

Comment:
I have information in the table. It was previously displaying just fine until i tried delete from php my admin. When i vardump i get the message NULL.